sycl/unittests/Extensions/CommandGraph.cpp

```diff
@@ -2178,3 +2178,65 @@ TEST_F(MultiThreadGraphTest, Finalize) {
     ASSERT_EQ(checkExecGraphSchedule(GraphExecImpl, GraphExecRefImpl), true);
   }
 }
+
+// Test adding fill and memset nodes to a graph
```

Comment:
FYI, AFAIK, we're not running unittests in CI. We have an internal bug report for this, but I don't know anyone actively working on it.

Reply:
That's good to know, I wasn't aware of that.

Reply:
One can run `check-sycl-unittests` locally if they wish :)